docs/running_rune_with_occlum.md

```diff
@@ -128,7 +141,7 @@ export OCCLUM_INSTANCE_DIR=occlum-app
 yum install -y libseccomp
 docker run -it --rm --runtime=rune \
   -e ENCLAVE_TYPE=intelSgx \
-  -e ENCLAVE_RUNTIME_PATH=/usr/lib/liberpal-occlum.so \
+  -e ENCLAVE_RUNTIME_PATH=/opt/occlum/build/lib/libocclum-pal.so.0.14.0 \
```

Comment:
/opt/occlum/build/lib/libocclum-pal.so.0.14.0 is installed by your package so it can be /opt/occlum/build/lib/libocclum-pal.so without version number right?